src/util.jsx

```diff
@@ -188,3 +188,15 @@ export function pxToUnit(dimensionString) {
   if (!dimensionString) { return 0; }
   return parseInt(dimensionString.slice(0, -2), 10);
 }
+
+export function joinClassNames(className1, className2) {
```

Comment:
We can probably just add `classnames` as a dependency, but I'll leave that decision to you